src/measure_theory/group/fundamental_domain.lean

```diff
@@ -202,6 +222,20 @@ by simpa only [set_lintegral_one] using h.set_lintegral_eq_tsum (λ _, 1) t
   μ t = 0 :=
 by simp [measure_eq_tsum h, ht, hts]
 
+/-- Given a measure space with an action of a finite group `G`, the measure of any `G`-invariant set
+is determined by the measure of its intersection with a fundamental domain for the action of `G`. -/
+@[to_additive measure_eq_card_smul_of_vadd_eq_self "Given a measure space with an action of a finite
+additive group `G`, the measure of any `G`-invariant set is determined by the measure of its
+intersection with a fundamental domain for the action of `G`."]
+lemma measure_eq_card_smul_of_smul_eq_self [finite G]
+  (h : is_fundamental_domain G s μ) (t : set α) (ht : ∀ g : G, g • t = t) :
```

Comment:
Is it easy to weaken the lemma by requiring in `ht` that `g • t` is almost everywhere equal to `t` instead? (Ignore if this is nontrivial).

Reply:
Good point. Now done (it was easy).